### PR TITLE
Update communication by Charm.md

### PR DESCRIPTION
**Pull request by MyatCharm**
---
name: solution review 
about: A template PR for code review with a checklist
---

### Files

- [ ] The file name describes the function's behavior
- [ ] There is a module docstring in the function file
- [ ] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [ ] There is a module docstring in the tests file

### Unit Tests

- [ ] The test class has a helpful name in PascalCase
- [ ] The test class has a docstring
- Every unit test has
  - [ ] A helpful name
  - [ ] A clear docstring
  - [ ] Only one assertion
  - [ ] There is no logic in the unit test
- [ ] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases

### Function Docstring

- [ ] The function's behavior is described
- The function's arguments are described:
  - [ ] Type
  - [ ] Purpose
  - [ ] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [ ] Type
  - [ ] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [ ] Each assumption about an argument is checked with an assertion
  - [ ] Each assertion checks for _only one_ assumption about the argument
- [ ] Include 3 or more (passing!) doctests

### The Function

- [ ] The function's name describes it's behavior
- [ ] The function's name matches the file name
- [ ] The function has correct type annotations
- [ ] The function is not called in the function file

### Do's

- [ ] Variable names help to understand the strategy
- [ ] Any comments are clear and describe the strategy
- [ ] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [ ] The function's strategy _is not_ described in the documentation
- [ ] Comments explain the _strategy_, **not** the _implementation_
- [ ] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [ ] The code passes the formatting checks
- [ ] The code passes all Ruff linting checks
- [ ] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [ ] Variables are named with snake_case
- [ ] Variable names are clear and helpful
- [ ] The code follows the strategy as simply as possible
- [ ] The implementation is as simple as possible given the strategy
- [ ] There are no commented lines of code
- [ ] There are no `print` statements anywhere
- [ ] The code includes defensive assertions
- [ ] Defensive assertions include as little logic as possible

### CODE REVIEW

**Comment by Raneemrami** [comment]
Hi,
have you tried adding a blank line after the( ### Availability for calling/messaging) heading, I think
you will pass the MD linting if you did

**Comment by Raneemrami** [comment]
Oh, Sorry, it is good now

**Comment by MyatCharm** [comment]
Yeah, I have fixed it in VS code. If you could merge after reviewing, you can continue develop your adding your available time after mine I suppose.

File: collaboration/communication.md [hunk 1]
```diff
@@ -37,17 +41,29 @@ ______________________________________________________________________
...
 
-- name: _5h_;
-- name: _6h_;
-- name: _5h_;
-- name: _4h_;
-- name: _3h_;
+- Muqadsa: _4-5h (flexible if needed)_;
+- Charm: _4-5h (flexible)_;
+- Raneem: _?h_;
+- Pyae: _?h_;
```
Comment (PyaeLinn01) [inline collaboration/communication.md]: 5h

File: collaboration/communication.md [hunk 2]
```diff
@@ -37,17 +41,29 @@ ______________________________________________________________________
...
 ### How many hours everyone has per day
 
-- name: _5h_;
-- name: _6h_;
-- name: _5h_;
-- name: _4h_;
-- name: _3h_;
+- Muqadsa: _4-5h (flexible if needed)_;
+- Charm: _4-5h (flexible)_;
+- Raneem: _?h_;
```
Comment (PyaeLinn01) [inline collaboration/communication.md]: @Raneemrami  you can comment yours.

File: collaboration/communication.md [hunk 3]
```diff
@@ -37,17 +41,29 @@ ______________________________________________________________________
...
+- Pyae: _?h_;
+- MayMon: _?h_;
```
Comment (PyaeLinn01) [inline collaboration/communication.md]: @MayMon-T3807 you can comment yours

File: collaboration/communication.md [hunk 4]
```diff
@@ -37,17 +41,29 @@ ______________________________________________________________________
...
+- Jubayer: _?h_;
```
Comment (PyaeLinn01) [inline collaboration/communication.md]: @MD-Jubayer-Khan you can comment yours

File: collaboration/communication.md [hunk 5]
```diff
@@ -37,17 +41,29 @@ ______________________________________________________________________
...
+- Thilakan: _?h_;
```
Comment (PyaeLinn01) [inline collaboration/communication.md:65]: @Akan186 you can comment yours

File: collaboration/communication.md [hunk 6]
```diff
@@ -37,17 +41,29 @@ ______________________________________________________________________
...
+- Safa: _?h_;
```
Comment (PyaeLinn01) [inline collaboration/communication.md:66]: @Safa-Saber you can comment yours

File: collaboration/communication.md [hunk 7]
```diff
@@ -37,17 +41,29 @@ ______________________________________________________________________
...
-| Day | Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday | |
------- | :----: | :-----: | :-------: | :------: | :----: | :------: | :----: |
-| _name_ | | | | | | | |
+(In terms of EST)
+
+|          | Mon | Tue | Wed | Thur | Fri | Sat | Sun |
+|----------|-----|-----|-----|------|-----|-----|-----|
+| Muqadsa  | ✓   | ✓   | ✓  | ✓    | ✓   | ✓   | ✓ |
+| Charm    | ✓   | ✓   | ✓  | ✓    | ✓   | ✓   | ✓ |
+| Raneem   | ?   | ✓   | ?   | ?    | ✓   | ✓   | ✓ |
```
Comment (PyaeLinn01) [inline collaboration/communication.md:50]: @Raneemrami you can comment yours

File: collaboration/communication.md (hunk 6 above)
Comment (Safa-Saber) [reply collaboration/communication.md:66]: Okay

File: collaboration/communication.md (hunk 4 above)
Comment (MD-Jubayer-Khan) [reply collaboration/communication.md]: From now I will be able to join anytime when others members are able to. so you can add some common slots as the others members

File: collaboration/communication.md (hunk 2 above)
Comment (Raneemrami) [reply collaboration/communication.md]: mine is (flexible)
I do not want to add much details,

File: collaboration/communication.md (hunk 3 above)
Comment (MayMon-T3807) [reply collaboration/communication.md]: 5 h

File: collaboration/communication.md (hunk 7 above)
Comment (Raneemrami) [reply collaboration/communication.md:50]: you can just keep it as it is, Thank You

Comment (PyaeLinn01) [reply collaboration/communication.md:50]: ok
